bundles/binding/org.openhab.binding.dsmr/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="dsmr"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>DSMR Binding</name>
+    <description>The DSMR Binding obtains data from a Dutch smart meter ("Slimme meter" in Dutch) via the P1-port.</description>
+
+    <config-description>
+        <parameter name="port" type="text" required="true">
+            <label>Serial port</label>
+            <description>The DSMR serial port, e.g. "COM1" for Windows or "/dev/ttyUSB0" for Linux.</description>
+            <default>/dev/ttyUSB0</default>
+        </parameter>
+    </config-description>
+</binding:binding>
```

Comment:
If the file should not be here then following page needs to be updated:

http://docs.openhab.org/developers/development/compatibilitylayer.html#how-to-use-openhab-1x-add-ons-that-are-not-part-of-the-distribution

> How to add a successfully tested 1.x Add-on to the distribution
> ...
> Note that with defining a Karaf feature, bindings are available for installation through the Paper UI in the “Extensions” menu, but they are not listed under “Configuration->Bindings” (although it is fully operational after installation). In order to have bindings listed there as well, you need to add some meta-information to the binding bundle. This information should be put into ESH-INF/binding/binding.xml and its content is described here. Do not forget to add ESH-INF to your build.properties, so that it is packaged in the bundle. See a real life example of such meta-data here.

Also 7 other OH1 bindings that use it this way need to be updated.

The compatibility page could also have some notes on the interactions that @watou asks for as well as do's and don'ts.

The main issue with the page is its discoverability though. I know it exists but even had problems finding it again. It could also help when the broken links in the [opening post](https://community.openhab.org/t/we-need-your-help-on-testing/298) in the community topic are updated.